pydantic/mypy.py

```diff
@@ -121,7 +121,7 @@ def get_method_hook(self, fullname: str) -> Optional[Callable[[MethodContext], T
 
     def get_class_decorator_hook(self, fullname: str) -> Optional[Callable[[ClassDefContext], None]]:
         if fullname == DATACLASS_FULLNAME:
-            return dataclasses.dataclass_class_maker_callback  # type: ignore[return-value]
+            return dataclasses.dataclass_class_maker_callback
```

Comment:
I think we don't need this change. you may have an old version of mypy in your local setup